mclagsyncd/mclaglink.h

```diff
@@ -90,36 +134,128 @@ struct mclag_fdb
 
 };
 
+
+//MCLAG Domain Key 
+struct mclagDomainEntry
+{
+   unsigned int domain_id;
+
+   mclagDomainEntry() {}
+   mclagDomainEntry(unsigned int id):domain_id(id) {}
+
+   bool operator <(const mclagDomainEntry &domain) const
+   {
+       return domain_id < domain.domain_id;
+   }
+
+   bool operator ==(const mclagDomainEntry &domain) const
+   {
+       if (domain_id != domain.domain_id)
+           return 0;
+       return 1;
+   }
+};
+
+
+//MCLAG Domain Data
+struct mclagDomainData
+{
+    std::string source_ip;
+    std::string peer_ip;
+    std::string peer_link;
+    int keepalive_interval;
+    int session_timeout;
+
+    mclagDomainData()
+    {
+        keepalive_interval = -1;
+        session_timeout    = -1;
+    }
+
+    bool mandatoryFieldsPresent() const
+    {
+        if (!source_ip.empty() && !peer_ip.empty())
+            return 1;
+        return 0;
+    }
+
+    bool allFieldsEmpty() const
+    {
+        if (source_ip.empty() && peer_ip.empty() && peer_link.empty()
```

Comment:
The same as above - you can just return this whole expression instead of checking

Reply:
addressed review comments

Reply:
@gitsabari Did you decide not to change it or just forgot to commit the changes?